hardware/computemodule/cm-emmc-flashing.md

```diff
@@ -118,6 +118,22 @@ The `/dev/sdX1` and `/dev/sdX2` partitions can now be mounted normally.
 
 Make sure J4 (USB SLAVE BOOT ENABLE) / J2 (nRPI_BOOT) is set to the disabled position and/or nothing is plugged into the USB slave port. Power cycling the IO board should now result in the Compute Module booting from eMMC.
 
+## Flashing the bootloader EEPROM - Compute Module 4
+The `rpiboot` tool is the recommended method for updating the bootloader EEPROM on Compute Module 4. After following the initial EMMC flashing setup steps run the following command to run the `recovery` image instead of the EMMC image.
+
+```bash
+./rpiboot -d recovery
+```
+
+`rpiboot` contains a default `pieeprom.bin` bootloader EEPROM image. See the [boot EEPROM](../raspberrypi/booteeprom.md) and [bootloader configuration](../raspberrypi/bcm2711_bootloader_config.md) pages for more information about how to change the embedded configuration file.
```

Comment:
What does "`rpiboot` contains a default `pieeprom.bin` bootloader EEPROM image." mean? I thought rpiboot is just the program that serves files to a ComputeModule that's booting in USB device mode?

Reply:
It includes the latest default version

Reply:
Ahhh, I guess I must have been confusing "the rpiboot executable" and "the rpiboot package" :wink:

Reply:
Yes, rpiboot it's rather confusing given that the Github repo is usbboot!

Essentially this is just bundled as a 'reset my devboard' option rather than having to clone the rpi-eeprom repo as well. For product development, we would encourage customers to verify a particular bootloader release + config meets there needs then freeze again that version and replace the image in their own local fork of this repo.